package.json

```diff
@@ -83,38 +83,35 @@
     "underscore.string": "^3.3.4"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
```

Comment:
Yay! (wanted this but thought of waiting till TS which allows this by default)

Reply:
@ranbena This syntax is already supported (at least I use it sometimes) - not sure which plugin does it, but...

Reply:
Show me a in the code!

Reply:
[For example](https://github.com/getredash/redash/blob/master/client/app/components/ParameterMappingInput.jsx#L119-L139)

Reply:
and BTW - it's even your code :smile:

Reply:
Right. I guess what confused me is that I'm used to defining these first in classes (props defined outside class, constructor ,method rarely used). [Like here](https://github.com/CityOfZion/neon-wallet/blob/dev/app/hocs/withCameraAvailability.js).

Reply:
To be honest, I follow the instruction below 🤔 
```
 PASS  client/app/components/Footer.test.js
 PASS  client/app/components/UserShow.test.js
 FAIL  client/app/components/queries/ScheduleDialog.test.js
  œ Test suite failed to run                                                                                                                          [        
    SyntaxError: /home/vagrant/go/src/github.com/koooge/redash/client/app/components/queries/ScheduleDialog.jsx: Support for the experimental syntax 'classPrope
rties' isn't currently enabled (23:20):

      21 |
      22 | export class ScheduleDialog extends React.Component {
    > 23 |   static propTypes = {
         |                    ^
      24 |     show: PropTypes.bool.isRequired,
      25 |     schedule: RefreshScheduleType,
      26 |     refreshOptions: PropTypes.arrayOf(PropTypes.number).isRequired,

    Add @babel/plugin-proposal-class-properties (https://git.io/vb4SL) to the 'plugins' section of your Babel config to enable transformation.

      at Object.raise (node_modules/@babel/parser/lib/index.js:3831:17)
      at Object.expectPlugin (node_modules/@babel/parser/lib/index.js:5148:18)
      at Object.parseClassProperty (node_modules/@babel/parser/lib/index.js:8229:12)
      at Object.pushClassProperty (node_modules/@babel/parser/lib/index.js:8193:30)
      at Object.parseClassMemberWithIsStatic (node_modules/@babel/parser/lib/index.js:8132:14)
      at Object.parseClassMember (node_modules/@babel/parser/lib/index.js:8069:10)
      at withTopicForbiddingContext (node_modules/@babel/parser/lib/index.js:8024:14)
      at Object.withTopicForbiddingContext (node_modules/@babel/parser/lib/index.js:7150:14)
      at Object.parseClassBody (node_modules/@babel/parser/lib/index.js:8001:10)
      at Object.parseClass (node_modules/@babel/parser/lib/index.js:7975:10)
```

Reply:
That's weird. I don't get that error..
@kravets-levko any idea what's making this syntax valid without the plugin?

Reply:
nodejs version :slightly_smiling_face:

Reply:
So we should upgrade node from the currently defined v8.0.0?

Reply:
I currently use `v10.13.0`, but `v8.0.0` seems to work fine for everything except of tests. Not sure what we should change... :thinking: Is it possible to setup `jest` to use babel for loading test files?

Reply:
The test failed with the same message in my environment with node 8.15.0 which is no @babel/plugin-proposal-class-properties.
My guess is the test passes because the master branch has babel-preset-stage2. 
https://github.com/tc39/proposal-class-public-fields

Reply:
I'm not sure what's the best practice here, but maybe we should setup Jest to run on code post webpack compilation? It might slow down tests, so maybe there are better solutions?

Reply:
My understanding is that class-properties is still experimental stage https://github.com/tc39/proposal-class-public-fields.
After all, `npm test` and `npm build` need @babel/plugin-proposal-class-properties in babel7 instead of npm-preset-stage2 in babel6.